core/include/webview/detail/backends/cocoa_webkit.hh

```diff
@@ -157,8 +130,7 @@ public:
       m_window_delegate = nullptr;
     }
     if (m_app_delegate) {
-      auto app = get_shared_application();
-      objc::msg_send<void>(app, "setDelegate:"_sel, nullptr);
+      objc::msg_send<void>(m_app, "setDelegate:"_sel, nullptr);
       // Make sure to release the delegate we created.
```

Comment:
I favoured initialising `m_app` in the class constructor instead of repeated calls to `get_shared_application`